src/snowflake/snowpark/functions.py

```diff
@@ -5775,32 +5775,34 @@ def vector_cosine_distance(v1: ColumnOrName, v2: ColumnOrName) -> Column:
     """Returns the cosine distance between two vectors of equal dimension and element type.
 
     Example::
-        >> from snowflake.snowpark.functions import vector_cosine_distance
-        >> df = session.sql("select [1,2,3]::vector(int,3) as a, [2,3,4]::vector(int,3) as b")
-        >> df.select(vector_cosine_distance(df.a, df.b).as_("dist")).show()
+        >>> from snowflake.snowpark.functions import vector_cosine_distance
+        >>> df = session.sql("select [1,2,3]::vector(int,3) as a, [2,3,4]::vector(int,3) as b")
+        >>> df.select(vector_cosine_distance(df.a, df.b).as_("dist")).show()
         ----------------------
         |"DIST"              |
         ----------------------
         |0.9925833339709303  |
         ----------------------
+        <BLANKLINE>
     """
     v1 = _to_col_if_str(v1, "vector_cosine_distance")
     v2 = _to_col_if_str(v2, "vector_cosine_distance")
     return builtin("vector_cosine_distance")(v1, v2)
 
 
 def vector_l2_distance(v1: ColumnOrName, v2: ColumnOrName) -> Column:
-    """Returns the cosine distance between two vectors of equal dimension and element type.
+    """Returns the l2 distance between two vectors of equal dimension and element type.
 
     Example::
-        >> from snowflake.snowpark.functions import vector_l2_distance
-        >> df = session.sql("select [1,2,3]::vector(int,3) as a, [2,3,4]::vector(int,3) as b")
-        >> df.select(vector_l2_distance(df.a, df.b).as_("dist")).show()
+        >>> from snowflake.snowpark.functions import vector_l2_distance
```

Comment:
For my learning, what's the purpose of three >?

Reply:
it makes it an actual doc string example code, three > give those lines code block style in the doc